org.eclipse.xtext/src/org/eclipse/xtext/Xtext.xtext

```diff
@@ -44,7 +44,7 @@ Annotation returns Annotation:
 ;
 
 ParserRule :
-	(annotations += Annotation)?
+	(annotations += Annotation)*
```

Comment:
We need to adjust this for TerminalRule, too

Reply:
Done.